src/cljc/rems/text.cljc

```diff
@@ -49,8 +50,13 @@
         (first (vals m)))))
 
 ;; TODO: replace usages of `get-localized-title` with `localized`
-(defn get-localized-title [item language]
+;;
+;; TODO: remove fallback that fetches :title from item: localizations
+;;;  could be forced to always have the title in either the language of
+;;   choice or the default language.
+(defn get-localized-title [item language default-language]
   (or (get-in item [:localizations language :title])
+      (get-in item [:localizations default-language :title])
       (:title item)))
```

Comment:
The admin UI should already force translations for all languages to be filled in, so all these fallbacks should be unnecessary.

Reply:
Yeah, I started thinking along the same lines after your previous comment. The problem is that at least LBR has some missing Finnish translations, but maybe I'll just copy the English translations to fill them in, in the migrations.

Reply:
A more reliable fallback, in case the item is missing the requested translation, is to randomly pick whatever translation the item does have.

Reply:
It's possible to have a situation that the default language is English, but the item has only Finnish translation.

Reply:
True, I'll use that instead.

Reply:
I ended up using just the fallback to care for missing Finnish translations for catalogue items in the LBR database and not add extra logic to migrations. LBR can probably fill in the missing translations using the UI once the catalogue items are editable.